src/Controls/src/Core/Handlers/Items/Android/ItemContentView.cs

```diff
@@ -127,8 +149,7 @@ protected override void OnMeasure(int widthMeasureSpec, int heightMeasureSpec)
 				pixelHeight = (int)this.ToPixels(measure.Height);
 			}
 
-			_reportMeasure?.Invoke(new Size(pixelWidth, pixelHeight));
-			_reportMeasure = null; // Make sure we only report back the measure once
```

Comment:
We can't really set this to null because if the constraints on the CV change we need to report a new measure.